string saving_activation
int64 iteration_nb
---
string saving_activation_response